spec/menu_spec.rb

```diff
@@ -0,0 +1,38 @@
+require_relative "../lib/menu"
+
+describe Menu do
+  let(:menu) { Menu.new }
+  # let(:dishes) { {"carbonara" => 3.20, "polpette" => 4.50} }
+  # let(:order) { Menu.new.choose }
+
+  it { is_expected.to respond_to(:view) }
```

Comment:
all of these methods are being tested on later. Is there a purpose to responding to them even thought they get tested?